### PR TITLE
fix(files): [#FOR-797] fix files responses

### PR DESCRIPTION
**Pull request by qperie**
## Describe your changes
Fix saving errors when responding to a type file question

Documentation : 
- https://confluence.support-ent.fr/display/FOR/Response#operations-tag-default
- https://confluence.support-ent.fr/display/FOR/Response+File#operations-tag-default

## Issue ticket number and link
FOr-797 : https://jira.support-ent.fr/browse/FOR-797

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: common/src/main/resources/ts/models/Files.ts [hunk 1]
```diff
@@ -8,25 +8,24 @@ export class Files {
...
-            }
-            let file = new FormData();
-            file.append(Constants.FILE, this.all[i], filename);
-        }
-    }
 }
 
 export class FilePayload {
-    file: File;
+    formData: FormData;
```
Comment (qperie) [inline common/src/main/resources/ts/models/Files.ts:14]: L'erreur majeure du ticket : on veut des FormData et non pas des File

File: common/src/main/resources/ts/services/ResponseFileService.ts [hunk 2]
```diff
@@ -80,7 +80,7 @@ export const responseFileService: ResponseFileService = {
 
     async deleteAllMultiple(responseIds: number[]) : Promise<void> {
         try {
-            return DataUtils.getData(await http.delete(`/formulaire/responses/files`));
+            return DataUtils.getData(await http.delete(`/formulaire/responses/files/multiple`, { data: responseIds }));
```
Comment (qperie) [inline common/src/main/resources/ts/services/ResponseFileService.ts:83]: Renommée pour éviter le conflit avec une autre route API

**Review by qpic-cgi — APPROVED** [review]

File: formulaire/src/main/java/fr/openent/formulaire/controllers/ResponseFileController.java [hunk 3]
```diff
@@ -252,7 +252,7 @@ public void deleteAll(HttpServerRequest request) {
         deleteAllByResponse(request, responseIds);
     }
 
-    @Delete("/responses/files")
+    @Delete("/responses/files/multiple")
```
Comment (BastienBenard) [inline formulaire/src/main/java/fr/openent/formulaire/controllers/ResponseFileController.java:255]: LA doc d'api a était changé ?

Comment (qperie) [reply formulaire/src/main/java/fr/openent/formulaire/controllers/ResponseFileController.java:255]: Pas encore, j'suis en mode urgence mais une fois en recette je prendrai le temps de la modifier yep, y'en a plusieurs qui ont légèrement changé en plus 👍

**Pull request merged**